app/src/main/java/org/schabi/newpipe/streams/Mp4FromDashWriter.java

```diff
@@ -640,19 +640,20 @@ private int makeFtyp() throws IOException {
         return size;
     }
 
-    private byte[] makeMdat(long refSize, final boolean is64) {
+    private byte[] makeMdat(final long refSize, final boolean is64) {
```

Comment:
Why even change this file? That's unrelated to your PR and not needed. Plus if you're gonna fix a suppressed error, at least remove it from the suppressions.

Reply:
My build could not succeed and the log prints out that it should be changed to "final" variable because of the CheckStyle. That's why I fixed it.

Reply:
@ludugz: That's actually mpossible afaik, since line 643 is suppressed, as you could see [here](https://github.com/TeamNewPipe/NewPipe/blob/dev/checkstyle-suppressions.xml#L19).

Reply:
@wb9688 Yeah you are right. I don't know why but this time I build without any problem related to this, I will put this file back as before.